code/game/area/BigRed.dm

```diff
@@ -340,8 +340,8 @@
 	icon_state = "blueold"
 	ceiling = CEILING_METAL
 
-/area/bigredv2/outside/marshall_office
-	name = "\improper Marshall Offices"
+/area/bigredv2/outside/marshal_office
```

Comment:
This will require you to rename the areas in the map, else it will render it unusable.

Reply:
Should be resolved now, along with the CO jumpsuit. Didn't touch desert-dam since it didn't reference that area when I opened it.